docs/guidelines/content-guidelines/10-docs-preview.md

```diff
@@ -1,43 +1,29 @@
 # Documentation Preview
 
-Before you publish new documentation on the Kyma website, you can preview your changes to see if the formatting of the text is correct, images fit well, and links work as expected.
+Before you publish new documentation on the Kyma website, you can preview your changes to see if the formatting of the text is correct, images fit well, and links work as expected. To do that, see the following steps:
 
-That is possible thanks to the [**docsify preview feature**](https://docsify.js.org/#/quickstart?id=preview-your-site) supported by [docsify](https://docsify.js.org/#/).
+1. Go to the [`kyma/hack`](https://github.com/kyma-project/kyma/tree/main/hack), and execute the `copy_external_content.sh` script:
```

Comment:
Do the files in the `hack` folder also need to be updated once we add a new module repo? For example, https://github.com/kyma-project/kyma/blob/main/hack/copy_external_content.sh?